src/App/ComponentsDocumentation/components/Forms/index.js

```diff
@@ -16,32 +17,45 @@ const Overview = () => (
             show a collection of related inputs. <CodeTags type="primary" code={"<form-group>"} />s can contain more than just an input, like a <CodeTags type="primary" code={"<label>"} /> or
             <CodeTags type="primary" code={"<help-block>"} />s.
         </p>
+        <p>
+            <b>Note:</b> On larger screens, <CodeTags type="primary" code={"<input>"} /> should not be wider than half the
+            page. <CodeTags type="primary" code={"<input>"} /> should take the available width on smaller screen sizes. <Link to="/core/grid">Grid</Link> can
+            be used to achieve this behavior.
```

Comment:
The following sections should probably follow these guidelines: "Validation overview", "Feedback", and "Dynamically disabled submit button",